x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/onboarding_card.tsx

```diff
@@ -69,7 +69,7 @@ export const OnboardingCard: React.FC<OnboardingCardProps> = ({
       <EuiPanel color="subdued" hasShadow={false}>
         <EuiEmptyPrompt
           iconType={complete ? 'checkInCircleFilled' : (icon as IconType)}
-          iconColor={complete ? 'secondary' : 'subdued'}
+          iconColor={complete ? 'success' : 'subdued'}
```

Comment:
should `subdued` be changed to `text` here?

Reply:
That change is only necessary for `EuiButtonIcon`, so this update is correct